app/models/job.rb

```diff
@@ -32,4 +30,10 @@ def self.total_pages(num_of_items_per_page)
     calculation = last_two_months.count / num_of_items_per_page.to_f
     calculation.ceil
   end
+
+  def self.current_openings_technology_count
+    Technology.all.map do |tech|
+      {label: tech.name, value: last_two_months.by_tech(tech.name).count}
```

Comment:
Probably would want to build a single query to avoid N + 1 queries here.  For now, we can stick with this though